gapid_tests/command_buffer_tests/BeginAndEndRenderPass_test/BeginAndEndRenderPass_test.py

```diff
@@ -100,13 +100,6 @@ def expect(self):
         require_equal(0, begin_render_pass_info.clearValueCount)
         require_equal(0, begin_render_pass_info.pClearValues)
 
-        clear_values = VulkanStruct(
-            architecture, CLEAR_COLOR_UNORM_VALUE, get_read_offset_function(
-                begin_render_pass, begin_render_pass_info.pClearValues))
-        # 0.2f == 0x3e4ccccd
-        expected_color_clear_value = [0x3e4ccccd for i in range(4)]
```

Comment:
Nope, pClearValues was changed to be null, so there is no read observation at all.